src/parcels/parcels.module.ts

```diff
@@ -3,14 +3,13 @@ import { ParcelsController } from './parcels.controller';
 import { ParcelsService } from './parcels.service';
 import {parcelProviders, parcelTrackingProviders} from './parcel.providers';
 import { DatabaseModule } from '../db/database.modules';
-import { UsersModule } from '../users/users.module';
 import {pushTokenProvider} from '../push-token/push-token.providers';
 import {PushTokenService} from '../push-token/push-token.service';
 
 @Module({
-  imports: [DatabaseModule, UsersModule],
+  imports: [DatabaseModule],
```

Comment:
it doesn't look like ParcelsModule was using the UsersModule, so I removed it to avoid circular dependencies since UsersModule is now importing ParcelsModule.

Reply:
Good catch - thank you